internal/sirius/document.go

```diff
@@ -0,0 +1,121 @@
+package sirius
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+type Document struct {
+	ID              int    `json:"id,omitempty"`
+	CorrespondentID int    `json:"correspondentID"`
+	Type            string `json:"type"`
+	SystemType      string `json:"systemType"`
+	FileName        string `json:"fileName,omitempty"`
+	Content         string `json:"content"`
+}
+
+type documentTemplateApiResponse map[string]json.RawMessage
+type insertApiResponse map[string]json.RawMessage
+
+type UniversalTemplateData struct {
```

Comment:
sorry I didn't pick up on this before, but I think repeating these two fields in each struct makes more sense than embedding this struct within them

Reply:
I've kept the `UniversalTemplateData` for the deserialisation in `toDocumentData` and `toInsertData` but i have repeated the fields for the other structs